File: Backend.DPI/Backend.DPI/Models/RolPrivilege.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Backend.DPI.Models
{
    public partial class RolPrivilege
    {
        public int IdRolPrivilege { get; set; }
        public int PrivilegeIdPrivilege { get; set; }
        public int RolIdRol { get; set; }

        public virtual Privilege PrivilegeIdPrivilegeNavigation { get; set; }
        public virtual Rol RolIdRolNavigation { get; set; }
    }
}

File: Frontend.DPI/Frontend/src/app/rol_privileges/interfaces/rol-privilege.ts
export interface RolPrivilege{
    idRolPrivilege: number,
    name_Rol: string,
    name_Privilege: string
};

export interface CreateRolPrivilege{
    privilegeIdPrivilege: number,
    rolIdRol: number
}

export interface Privilege{
    idPrivilege: number,
    name: string
}

export interface Rol{
    idRol: number,
    name: string
}